### DISCUSSION TITLE
Hive mind plugin (share ideas globally)

### DISCUSSION THREAD

**Discussion by jespino** [mattermost/community#131] [General Ideas]
Add a plugin that allows to publicly record ideas and automatically share them with everybody, very much like this but inside mattermost, and a bit "fire and forget" then during hackathons we can take all that hive-mind ideas and use them.

**Comment by esethna** [comment]
Like th eidea, but curious how this is much different from the "Feature Proposals" channel? Or is it just less "formal"?

**Reply by jespino** [reply]
I don't know, probably a voting system and less time based problems like getting things lost in the history

**Reply by esethna** [reply]
Got it, yes a voting system is a key differentiator

**Comment by Willyfrog** [comment]
could this be done via focalboard? might be better suited for it
